index.html

```diff
@@ -80,7 +80,7 @@ <h1 class="title"><span class="fire"><span>🔥</span>Fire</span>select</h1>
         </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/vue/1.0.21/vue.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/vue/1.0.21/vue.min.js"></script>
```

Comment:
i think the ideal here would be simply removing protocol: `//...` because github pages works on both protocols.